### DISCUSSION TITLE
Have anyone the same problem?

### DISCUSSION THREAD

**Discussion by Fluppe1** [omkarcloud/google-maps-scraper#109] [General]
![image](https://github.com/omkarcloud/google-maps-scraper/assets/131346661/1105c536-3b33-40c7-9683-2a1a21f6a768)

**Comment by Chetan11-dev** [comment]
Thank you, @Fluppe1, for creating the discussion and using our Google Maps Scraper Tool. Also, this bug was caused due to a faulty release, kindly re download the pro version from the Google Maps Scraper Pro Download Link (See your email).

**Comment by Fluppe1** [comment]
Perfect. I can now generate a CSV File. Now I have a problem that I already had before. It only generates 20 leads for me, although I have entered 120. Here are the pictures: 
![image](https://github.com/omkarcloud/google-maps-scraper/assets/131346661/54e6bcf2-f9c9-43da-810b-ae1ab3a83050)
![image](https://github.com/omkarcloud/google-maps-scraper/assets/131346661/0f5badcf-3894-4a45-a985-1d6f2b278d7c)

**Comment by Chetan11-dev** [comment]
Number of fields depend on Location. because Google only showed 20 result, we only scraped 20. In case you want more data, I suggest using Countries method as told in https://github.com/omkarcloud/google-maps-scraper#-how-to-scrape-all-cities-in-my-country
